### ISSUE TITLE
No information provided to screen readers about tooltips

### ISSUE THREAD

**Issue by paulGeoghegan** [shoelace-style/webawesome-alpha#11]
On the example page for tool tips, there is a button that you can hover over to make a tool tip appear, but it is not announced by NVDA.

**Comment by KonnorRogers** [comment]
@paulGeoghegan 

Assuming you're using chrome as well?

**Comment by paulGeoghegan** [comment]
No FF but I checked in Chrome and the same thing happens. If I arrow up then I can read the tool tip but it should be announced to the screen reader or at least indicate how to get to it because currently nothing seems to happen.

**Comment by KonnorRogers** [comment]
🤔 @claviska we may have to do the special case we talked about with tooltips, or build the tooltip into the component because of cross root aria issues.

@paulGeoghegan im working on getting a windows box to get test this NVDA stuff. I'm generally testing in VoiceOver, but recognize majority of screen reader users are on Windows + NVDA

**Comment by paulGeoghegan** [comment]
@KonnorRogers let me know if you have any other questions as I'm more than happy to help out. I'm actually a blind dev so I use NVDA all day.

**Comment by KonnorRogers** [comment]
> @KonnorRogers let me know if you have any other questions as I'm more than happy to help out. I'm actually a blind dev so I use NVDA all day.

Much appreciated thank you so much for the offer! If there's anything else we can do to help you please let us know as well!

**Comment by paulGeoghegan** [comment]
@KonnorRogers It's no problem. I only went through a few of the components today but I'll try get through the rest tomorrow and let you know if I find anything else.
I really do appreciate the work all of you are doing and that's exactly why I wanted to be involved in the Alpha so that in a years time when probably thousands of websites use these components that everyone can use them.
